internal/network/manager.go

```diff
@@ -158,12 +140,14 @@ func (m *Manager) Start(isOffer bool,
 	// Check the fingerprint if a certificate was exchanged
 	cert := dtlsConn.RemoteCertificate()
 	if cert != nil {
-		hashAlgo, err := dtls.HashAlgorithmString(fingerprintHash)
+		var hashAlgo dtls.HashAlgorithm
+		hashAlgo, err = dtls.HashAlgorithmString(fingerprintHash)
```

Comment:
(major style nit) I assume the linter complained about shadowing?

I stopped to parse this a few times, assuming there was something special going on here. Might be worth putting a lambda or something, just so it doesn't stand out.